### PR TITLE
Add depdendency on fdisk now that it is no longer Essential.

### PR DESCRIPTION
**Pull request by mwhudson**
For https://bugs.launchpad.net/ubuntu/+source/ubuntu-image/+bug/1877485

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -5,6 +5,7 @@ Maintainer: Lukasz 'sil2100' Zemczak <lukasz.zemczak@ubuntu.com>
 Build-Depends: debhelper (>= 9),
                dh-python,
                dosfstools,
+               sfdisk,
```
Comment (sil2100) [inline debian/control:8]: Oh no, typo! I'll fix that tho.

**Comment by sil2100** [comment]
I'll merge this (and cherry-pick to groovy) once at least one ADT test finishes.

**Comment by sil2100** [comment]
Ah, of course, the fdisk package did not exist in xenial (sfdisk was coming from util-linux back then). Need to figure out a way to get this merged so that we can still get xenial ADT testing on master done.

**Comment by vorlonofportland** [comment]
> Ah, of course, the fdisk package did not exist in xenial (sfdisk was coming from util-linux back then). Need to figure out a way to get this merged so that we can still get xenial ADT testing on master done.

'Depends: fdisk | util-linux (<< $version)'  should do the job.

**Comment by mwhudson** [comment]
I don't think the snap.sh failures are my fault...

**Pull request merged**